OpenRA.Mods.Common/Traits/Upgrades/UpgradeManager.cs

```diff
@@ -67,20 +74,17 @@ class UpgradeState
 		}
 
 		readonly List<TimedUpgrade> timedUpgrades = new List<TimedUpgrade>();
-		readonly Lazy<Dictionary<string, UpgradeState>> upgrades;
 		readonly Dictionary<IUpgradable, int> levels = new Dictionary<IUpgradable, int>();
+		Dictionary<string, UpgradeState> upgrades;
```

Comment:
Why not `readonly Dictionary<string, UpgradeState> upgrades = new Dictionary<string, UpgradeState>();`?